frontend/device/input.lua

```diff
@@ -258,10 +259,21 @@ function Input:handleKeyBoardEv(ev)
         end
     end
 
-    if ev.value == EVENT_VALUE_KEY_RELEASE then
-        if keycode == "Light" then
-            return keycode
-        elseif keycode == "Power" then
+    if ev.value == EVENT_VALUE_KEY_RELEASE and keycode == "Light" then
+        return keycode
+    end
+    
+    if keycode == "Power" then
+        local UIManager = require("ui/uimanager")
+        if ev.value == EVENT_VALUE_KEY_PRESS then
+            power_up_trigerred = false
```

Comment:
all variables need to be declared as local. If you want to reference it later in KEY_RELEASE event, you can set this variable and the `power_press_long` function as an attribute of Input. Our naming convention is `under_score` for variable name and `camelCase` for function name.